src/lib/functions/server.js

```diff
@@ -1,14 +1,12 @@
 const bodyParser = require('body-parser')
 const jwtDecode = require('jwt-decode')
-const lambdaLocal = require('lambda-local')
```

Comment:
`lambda-local` is now contained within the JS runtime. The server is runtime-agnostic.